media/css/firefox/developer-quantum-whatsnew.scss

```diff
@@ -0,0 +1,181 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import
+    '../pebbles/includes/lib';
+
+$base-blue: #306ffd;
+
+@mixin section-curve {
+    position: absolute;
+    display: block;
+    left: 0;
+    content: '';
+    background-position: center bottom;
+    background-repeat: no-repeat;
+    width: 100%;
+    visibility: visible;
+    z-index: 10;
+}
+
+//*------------------------------------------------------------------*/
+// Intro
+.section-intro {
+    background-color: #fff;
+    background-position: center bottom;
+    background-repeat: no-repeat;
+    text-align: center;
+
+    @media #{$mq-tablet} {
+        @include background-size(1440px 1034px);
+        background-image: url('/media/img/firefox/products/developer/quantum/whatsnew-hero-background.png');
+        background-position: 0 0;
```

Comment:
I think this should be centered horizontally (`background-position: center bottom;`)

Also in any window wider than 1440px the "blobs" are cut off with a hard edge. I made a background that blended out to white for the product page and I can send you a version without the blue wave on the bottom to use here.